### ISSUE TITLE
Ability to toggle annotation buttons

### ISSUE THREAD

**Issue by samidabeast** [blacklabel/annotations#75]
Right now if I want to update the chart to not show the annotation buttons, it does not do anything. Can this be added?

Example of it not working with button calling chart.update();
[http://jsfiddle.net/7ddsadLg/](http://jsfiddle.net/7ddsadLg/)

**Comment by pawelfus** [comment]
Thank you for reporting!

At this moment, `chart.update()` doesn't change anything in the annotations plugin. Right not to hide button you can hide manually each button:

```javascript
  chart.annotations.buttons.forEach(function (button) {
  		button[0].hide();
  });
```
Demo: http://jsfiddle.net/7ddsadLg/1/

**Comment by samidabeast** [comment]
Thank you for the quick response. This workaround works. However, if you start the chart with the buttons not enabled, they never show up. 

Example: [http://jsfiddle.net/7ddsadLg/2/](http://jsfiddle.net/7ddsadLg/2/)

**Comment by pawelfus** [comment]
This is correct, buttons must be rendered so we can hide them. You can use `chart.events.load` callback to hide them on init.
